src/index.ts

```diff
@@ -5,6 +5,13 @@ import { version } from '../package.json'
 import { getContextOrUser } from './utils'
 export { useLDReady, useLDFlag, ldInit, useLDClient } from './hooks'
 
+// Export required types from the base SDK.
+export type {
```

Comment:
Without this a consumer would have to import these types from the base package, which can cause some mismatching.